build/README.md

```diff
@@ -50,8 +50,12 @@ All Docker names are suffixed with a hash derived from the file path (to allow c
 ## Build artifacts
 The build system output all its products to a top level directory in the source repository named `_output`.
 These include the binary compiled packages (e.g. kubectl, kube-scheduler etc.) and archived Docker images.
-If you intend to run a component with a docker image you will need to import it from this directory with
-the appropriate command (e.g. `docker import _output/release-images/amd64/kube-scheduler.tar k8s.io/kube-scheduler:$(git describe)`).
+If you intend to run a component with a docker image you will need to load it from this directory with
+the appropriate command, e.g.
+```
+$ docker load --input _output/release-images/amd64/kube-controller-manager.tar
+Loaded image: registry.k8s.io/kube-controller-manager-amd64:v1.25.16-dirty
```

Comment:
also probably we should just show the command? that way it can be copy pasted, none of our other examples include the prompt (`$`) or any command output?

Reply:
Done! Apologies for the delay, I've been on PTO.